src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,143 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationServiceImpl registrationService =
+            new RegistrationServiceImpl();
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    //Valid users
+    private static final User VALID_USER_ONE = new User();
+    private static final User VALID_USER_TWO = new User();
+    private static final User VALID_USER_THREE = new User();
```

Comment:
one valid user is enough. 

instantiate it using `@beforeeach or @BeforeAll` or locally in test method (if it is used once)